services/migration/README.md

```diff
@@ -0,0 +1,4 @@
+# migration service
```

Comment:
could you also add a Changelog file?

Reply:
I thought about adding a CHANGELOG per service but I decided to do it at repo level first. If that works, we could think in extending it per service and package